api/src/models/sale-offer.ts

```diff
@@ -1,29 +1,15 @@
 import { BaseModel } from "./base";
-import { KittyItem } from "./kitty-item";
 
 class SaleOffer extends BaseModel {
-  id?: string;
+  saleItemId!: number;
+  saleItemType!: number;
+  saleItemCollection!: string;
+  transactionId!: string;
   price!: number;
```

Comment:
No more seller address?

Reply:
`saleItemCollection` is the seller address. Named the same as the variable in the contract.